scss/_root.scss

```diff
@@ -1,3 +1,9 @@
+@mixin cssvar($name, $value) {
+  @if $value != null {
```

Comment:
it's invalid because css var can be `null` too, it's can be useful to reset var, better solution:
~~~scss
@mixin cssvar($name, $value: false) {
  @if $value != false {
      @if $value == null {
        $value: initial;
      }
      --#{$prefix}#{$name}: #{$value};
  }
}
~~~